install/tools/man/ipa-idrange-fix.1

```diff
@@ -0,0 +1,111 @@
+.\"
+.\" Copyright (C) 2024  FreeIPA Contributors see COPYING for license
+.\"
+.TH "ipa-idrange-fix" "1" "May 26 2024" "IPA" "IPA Manual Pages"
+.SH "NAME"
+ipa\-idrange\-fix \- Analyse and fix IPA ID ranges
+.SH "SYNOPSIS"
+ipa\-idrange\-fix [options]
+.SH "DESCRIPTION"
+
+\fIipa-idrange-fix\fR is a tool for analysis of existing IPA ranges, users and 
+groups outside of those ranges, and functionality to propose and apply 
+remediations to make sure as much users and groups as possible end up in the 
+IPA-managed ranges. Before any changes are applied, full backup of the system
+is \fBSTRONGLY RECOMMENDED\fR.
+
+Do not use this program in unattended mode unless you are absolutely sure
+you are consenting to the tool's propositions.
+
+You can apply the propositions manually via \fIipa idrange(1)\fR commands.
+
+This tool requires to be run from \fBroot\fR, and does not require a kerberos
```

Comment:
s/from/as
You don't need a comma betwee "root" and "and".

Reply:
fixed